src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs

```diff
@@ -191,12 +192,15 @@ public static class PublishedContentExtensions
                 throw new ArgumentException(string.Format("The instance parameter does not implement Type '{0}'", type.Name), "instance");
             }
 
+            // Get the accessor for UmbracoContext & ApplicationContext
+            var umbracoApplicationContextAccessor = (IUmbracoApplicationContextAccessor)Activator.CreateInstance(Ditto.GetUmbracoApplicationContextAccessorType());
```

Comment:
@JimBobSquarePants Cool, done - in comment fd44882c2ac5ed5c9cdb68ec25829286278a0eda 👍